extensions/2.0/Vendor/EXT_mesh_features/README.md

```diff
@@ -267,6 +271,32 @@ Empty feature IDs (e.g. `{}`) are disallowed — a feature ID must explicitly se
 > }
 > ```
 
+#### Resolving Feature IDs Externally
+
+Feature IDs do not have to be associated with a property table. It is also possible to specify feature IDs to only identify the features in an asset, and then use their IDs in an application-specific manner. 
+
+> **Example:** A minimal example for assigning feature IDs to a mesh primitive, using an implicit vertex attribute: The default value for the `offset` is 0, and the value for `repeat` is 3. This means that the vertices will receive the feature IDs `[0, 0, 0, 1, 1, 1, 2, 2, 2, ...]`, causing the vertices of each triangle to receive the same feature ID. 
+>
+> ```jsonc
+> // Primitive:
+> "extensions": {
+>   "EXT_mesh_features": {
+>     "featureIds": [
+>       { "repeat": 3 }
+>     ]
+>   }
+> }
+> ```
```

Comment:
With the illustration below (thanks!), I think it'd be fine to omit the example JSON here. I was going to suggest simplifying to `{"attribute": 0}` but it doesn't really add new information that we haven't seen earlier.